### ISSUE TITLE
Little League constraints

### ISSUE THREAD

**Issue by DukeCityCowboy** [pvpoke/pvpoke#163]
I went through and weeded out ineligible pokemon from the Little League list and came up with this custom output. It basically removes the mythical and legendary pokemon with the exceptions of the released listed shadow legendaries, meltan/melmetal, and the lake trio (...and shadow mawile) since they can all theoretically be obtained at any level. The exclusions are mainly all of the pokemon that have minimum levels due to the restricted means they can be obtained, whether it's level 20 for raids and eggs, or level 15 for research reward/breakthroughs/PvP rewards/event encounters. Shadow Mawile being the only nonlegendary that could be obtained that could be eligible for which the purified or regular versions are not. ( read: can only be obtained at a minimum level for which even 0/0/0 IVs would surpass the 500cp maximum.) I did not see a listing for any pokemon whose movesets included "return" so I didn't bother going through to determine which purified pokemons' minimum CP surpasses the threshold but since the minimum level for purified pokemon is 25 and the minimum IV combination possible is 1/1/1 through a special trade (2/2/2 if untraded since they always gain 2 to their IVs) it could end up mattering like it did for shadow/purified wobbuffet if "return" comes up again. The only pokemon with minimum level restrictions I noted that are eligible, but only barely, and must be acquired through a trade, are Riolu, Mime Jr., and Klink. Klinks current default and maximumized IVs all put it over level 20 so they are fine but but its level range needs to be adjusted so that the minimum level Klink can be obtained (20) is reflected since I dont recall it ever being obtainable through a research pvp etc. Riolu's default values need to be adjusted so they dont populate a lvl 19 mon (I just zeroed out the attack IV  to barely generate a lvl 20 mon but that puts its defaults unfairly optimized compared to others in the bracket so it was a concern since lower cp scores are far more impacted by IVs and that kind of thing is more likely to skew the data and the rankings. I dont know what math is usually used to determime the defaults though so I figure I would just mention it. Mime Jr. is even more of an issue since only a handful of IVs make the cut and the trade cant be made with a best friend anymore; the highest IV score possible is 4 (and only in defense), with optimum wins on a 0/4/1 combination followed by 0/1/3,  0/3/1, 0/4/0 etc.  Any IVs better than those and Mime Jr. overshoots. The remaining are just a few overrides for these 3 pokemon as well as a few other pokemon for which Ive determined better movesets. 
tldr: If I had more time, I would have written a shorter post!

thanks yall

{"name":"custom","title":"Custom","include":[{"filterType":"id","name":"Species","values":["mesprit"]},{"filterType":"id","name":"Species","values":["uxie"]},{"filterType":"id","name":"Species","values":["azelf"]},{"filterType":"id","name":"Species","values":["mewtwo_shadow"]},{"filterType":"id","name":"Species","values":["ho_oh_shadow"]},{"filterType":"id","name":"Species","values":["mawile_shadow"]},{"filterType":"id","name":"Species","values":["articuno_shadow"]},{"filterType":"id","name":"Species","values":["zapdos_shadow"]},{"filterType":"id","name":"Species","values":["moltres_shadow"]},{"filterType":"id","name":"Species","values":["raikou_shadow"]},{"filterType":"id","name":"Species","values":["entei_shadow"]},{"filterType":"id","name":"Species","values":["suicune_shadow"]}],"exclude":[{"filterType":"tag","name":"Tag","values":["mega"]},{"filterType":"tag","name":"Tag","values":["legendary"]},{"filterType":"id","name":"Species","values":["deoxys_defense"]},{"filterType":"id","name":"Species","values":["deoxys"]},{"filterType":"id","name":"Species","values":["deoxys_speed"]},{"filterType":"id","name":"Species","values":["deoxys_attack"]},{"filterType":"id","name":"Species","values":["jirachi"]},{"filterType":"id","name":"Species","values":["victini"]},{"filterType":"id","name":"Species","values":["mew"]},{"filterType":"id","name":"Species","values":["meloetta"]},{"filterType":"id","name":"Species","values":["lucario"]},{"filterType":"id","name":"Species","values":["bonsly"]},{"filterType":"id","name":"Species","values":["luxio"]},{"filterType":"id","name":"Species","values":["luxray"]},{"filterType":"id","name":"Species","values":["scrafty"]},{"filterType":"id","name":"Species","values":["mandibuzz"]},{"filterType":"id","name":"Species","values":["marowak_alolan"]},{"filterType":"id","name":"Species","values":["spiritomb"]},{"filterType":"id","name":"Species","values":["slowpoke_galarian"]},{"filterType":"id","name":"Species","values":["vullaby"]},{"filterType":"id","name":"Species","values":["klang"]},{"filterType":"id","name":"Species","values":["klinklang"]},{"filterType":"id","name":"Species","values":["pawniard"]},{"filterType":"id","name":"Species","values":["bisharp"]},{"filterType":"id","name":"Species","values":["munchlax"]},{"filterType":"id","name":"Species","values":["darumaka_galarian"]},{"filterType":"id","name":"Species","values":["rapidash_galarian"]},{"filterType":"id","name":"Species","values":["celebi"]},{"filterType":"id","name":"Species","values":["conkeldurr"]},{"filterType":"id","name":"Species","values":["mantyke"]},{"filterType":"id","name":"Species","values":["gurdurr"]},{"filterType":"id","name":"Species","values":["darmanitan_galarian"]},{"filterType":"id","name":"Species","values":["slowbro_galarian"]},{"filterType":"id","name":"Species","values":["elekid"]},{"filterType":"id","name":"Species","values":["mr.rime"]},{"filterType":"id","name":"Species","values":["mr.mime_galarian"]},{"filterType":"id","name":"Species","values":["runerigus"]},{"filterType":"id","name":"Species","values":["braviary"]},{"filterType":"id","name":"Species","values":["rufflet"]},{"filterType":"id","name":"Species","values":["pangoro"]},{"filterType":"id","name":"Species","values":["persian_alolan"]},{"filterType":"id","name":"Species","values":["smoochum"]},{"filterType":"id","name":"Species","values":["archen"]},{"filterType":"id","name":"Species","values":["archeops"]},{"filterType":"id","name":"Species","values":["krookodile"]},{"filterType":"id","name":"Species","values":["pancham"]},{"filterType":"id","name":"Species","values":["rotom_wash"]},{"filterType":"id","name":"Species","values":["ponyta_galarian"]},{"filterType":"id","name":"Species","values":["ditto"]},{"filterType":"id","name":"Species","values":["shedinja"]},{"filterType":"id","name":"Species","values":["mienfoo"]},{"filterType":"id","name":"Species","values":["mienshao"]},{"filterType":"id","name":"Species","values":["weezing_galarian"]},{"filterType":"id","name":"Species","values":["raichu_alolan"]},{"filterType":"id","name":"Species","values":["meowstic_male"]},{"filterType":"id","name":"Species","values":["meowstic_female"]},{"filterType":"id","name":"Species","values":["timburr"]},{"filterType":"id","name":"Species","values":["linoone_galarian"]},{"filterType":"id","name":"Species","values":["obstagoon"]},{"filterType":"id","name":"Species","values":["mawile"]},{"filterType":"id","name":"Species","values":["magby"]}],"overrides":[{"speciesId":"ducklett","fastMove":"WATER_GUN","chargedMoves":["BUBBLE_BEAM","BRAVE_BIRD"]},{"speciesId":"medicham","fastMove":"COUNTER","chargedMoves":["POWER_UP_PUNCH","ICE_PUNCH"]},{"speciesId":"bidoof","fastMove":"TACKLE","chargedMoves":["SUPER_POWER","SHADOW_BALL"]},{"speciesId":"riolu","fastMove":"COUNTER","chargedMoves":["LOW_SWEEP","CROSS_CHOP"]},{"speciesId":"mime_jr","fastMove":"CONFUSION","chargedMoves":["PSYBEAM","PSYCHIC"]},{"speciesId":"klink","fastMove":"VOLT_SWITCH","chargedMoves":["DISCHARGE","ZAP_CANNON"]},{"speciesId":"lugia","fastMove":"DRAGON_TAIL","chargedMoves":["AEROBLAST","HYDRO_PUMP"]},{"speciesId":"ho_oh","fastMove":"INCINERATE","chargedMoves":["BRAVE_BIRD","EARTHQUAKE"]}],"league":500,"levelCap":50}

**Comment by pvpoke** [comment]
Thanks so much for putting together this list! I'll think on how I can implement this (there are some hard coded restrictions for other leagues like Great League but I'd like to make a more dynamic system for it).

A few notes on a couple of the Pokemon: Uxie, Azelf, and Mesprit do spawn very rarely in the wild, and Rufflet and Mawile have been available under 500 CP during a Safari Zone and GO Fest respectively.
